velox/vector/ComplexVector.h

```diff
@@ -194,6 +194,8 @@ class RowVector : public BaseVector {
 
   void validate(const VectorValidateOptions& options) const override;
 
+  void resize(vector_size_t newSize, bool setNotNull = false) override;
```

Comment:
Should setNotNull be true as that mirrors what we have in BaseVector ? This is a safer setting I feel for a rowVector.